### ISSUE TITLE
Easy way to view versioned SLSA

### ISSUE THREAD

**Issue by joshuagl** [slsa-framework/slsa#166]
Now that we have our first release, how do we make it easy for folks to refer to that version of SLSA. I'm imagining something like how documentation published on Read The Docs both shows the current version of the documentation (latest, stable, or a version number) and has a toggle to select a different version of the docs.
<img width="1441" alt="Screenshot 2021-09-17 at 14 52 25" src="https://user-images.githubusercontent.com/13888612/133794503-81982b36-1a32-48c5-981f-3b4cf7f1e82e.png">

**Comment by joshuagl** [comment]
For The Update Framework (TUF) I wrote a GitHub Action that publishes each tagged release of the specification to a versioned folder in the GitHub-pages branch of the repo (see it [here](https://github.com/theupdateframework/specification/blob/master/.github/workflows/release.yml)). We could do something similar just for the SLSA spec (levels + requirements) and generate some HTML to switch between versioned paths when viewing those pages?

**Comment by MarkLodato** [comment]
That would work! To me the big challenge is all the styling - HTML, CSS, Jekyll. Not my area of expertise. :-)

**Comment by MarkLodato** [comment]
Before we do this, we need to agree on what pages need versioning and which do not. Here's a first pass:

-  Content that doesn't need versioning (I think):
   -  /example
   -  /faq
   -  /index
   -  /getinvolved
   -  /roadmap
   -  /use-cases/...
-  SLSA levels spec that definitely needs versioning:
   -  /levels
   -  /requirements
   -  /threats
-  SLSA provenance, which is already versioned:
   -  /provenance/...

As for how to do this we need both a URL design and a UX design.

For the URL, I suspect we can do something like this:

- /levels → **/spec/v0.1/levels**
- /requirements → **/spec/v0.1/requirements**
- /threats → **/spec/v0.1/threats**
- **/spec/latest** → **/spec/v0.1** (or "stable")

Related: #195, which needs a URL strategy for translations. FWIW, Python does https://docs.python.org/3/, https://docs.python.org/3.10/, https://docs.python.org/es/3/, etc.

For the UX, that's an open question that @devmoran and Projects by IF can hopefully solve. We probably want some sort of graphical switches similar to Python. We currently manually maintain a list of links for provenance, but (a) that's ugly and (b) it won't work well for the spec, which has multiple pages.

**Comment by devmoran** [comment]
Aha! This topic was on my list.
Thanks for tagging me on this @MarkLodato, acknowledging. Will read through this properly and contribute asap.

**Comment by devmoran** [comment]
As discussed briefly in yesterday's sync.

We've got a url routing plan-of-action (cc @mdunbavan), and a way to display it on the front end. Once we've got a POC working on our fork I'll add a link here for us all to try and discuss.

We'll be following the format `/spec/v0.1/xxxx`, with the ability to switch globally between the version you are reading for /levels, /provenance, /threats and /requirements.
